developers/arbitrum-deploy.md

```diff
@@ -119,3 +148,132 @@ You may need significantly more RAM and CPU to validate all blocks with WASM. Yo
 to send transactions to generate new batches to
 be posted to Celestia!
 :::
+
+## Deploy an Arbitrum rollup to Mocha testnet
+
+<!-- markdownlint-disable MD033 -->
+<script setup>
+import constants from '/.vitepress/constants/constants.js'
+</script>
+
+This section covers deploying an Arbitrum Nitro rollup to
+[Mocha testnet](../nodes/mocha-testnet.md) using Celestia as DA.
+
+### Dependencies
+
+- The first section of this page
+- A fully synced and funded Mocha testnet [light node](../nodes/light-node.md) on **v0.13.1**
+  - [Mocha testnet faucet](../nodes/mocha-testnet.md#mocha-testnet-faucet)
+
+
+### Configuration
+
+Since the contracts deployed through the factories above are already configured to communicate with Blobstream, you now only have to configure your node accordingly,  so lets walk through [an example found in nitro-testnode](https://github.com/celestiaorg/nitro-testnode/blob/celestia-v2.3.1/scripts/config.ts#L223-L233).
+
+First of all, you will need to enable Celestia DA in your Arbitrum chain params, which [you can see an example of](https://github.com/celestiaorg/nitro-testnode/blob/celestia-v2.3.1/scripts/config.ts#L358).
+
+For the configuration, use the following:
```

Comment:
please leave a comment to verify contract address to avoid people misusing wrong addresses (protection against copy-pasta devs) 🙏

Reply:
https://github.com/celestiaorg/docs/pull/1490/commits/5fc88f321bc7c8ff47d30c9dd0d4e82713e26eaf